hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/config/MirrorImporterConfiguration.java

```diff
@@ -66,13 +66,16 @@
     private final MetricsExecutionInterceptor metricsExecutionInterceptor;
     private final AwsCredentialsProvider awsCredentialsProvider;
 
-    @Resource(name = "webServerStartStop")
+    @Autowired(required = false)
+    @Qualifier("webServerStartStop")
     private SmartLifecycle webServerStartStop;
 
     @PostConstruct
     void init() {
         // Start the web server ASAP so kubernetes liveness probe is up before long-running migrations
-        webServerStartStop.start();
+        if (webServerStartStop != null) {
```

Comment:
Should this include a running check?
```suggestion
        if (webServerStartStop != null && !webServerStartStop.isRunning()) {
```

Reply:
Based on offline discussion this doesn't seem to be needed

Reply:
I don't think this would come up, the `start()` comes later in the lifecycle, after bean instantiation (and postInit).